### PR TITLE
Small progress towards non-native builds

### PR DESCRIPTION
**Pull request by hadess**

### CODE REVIEW

**Comment by atodorov** [comment]
Can one of the admins verify this patch?

**Comment by hadess** [comment]
`Make sure -machine is passed to qemu` is untested as I still didn't manage to force this to do anything, but the other two seem to behave as expected.

File: src/pylorax/installer.py [hunk 1]
```diff
@@ -142,6 +142,17 @@ class QEMUInstall(object):
                  "aarch64": "qemu-system-aarch64",
                  "ppc64le": "qemu-system-ppc64"
                 }
+    COMPATIBLE_ARCHS = {"x86_64": [ "x86_64", "i386" ],
```
Comment (bcl) [inline src/pylorax/installer.py]: Small style thing, no need for spaces between `[` and the contents of the list. We've (I!) have been inconsistent about this, but no spaces looks nicer.

**Comment by bcl** [comment]
I've added some commits to this. I'm unable to actually get aarch64 to work on my x86_64 system but I think it's close? See pr #1108

**Comment by hadess** [comment]
Thanks for taking the time cleaning this up.

**Pull request closed without merging**